### PR TITLE
Add missing ruby versions to CI file

### PR DESCRIPTION
**Pull request by guillermoap**
On https://github.com/rootstrap/rails_api_base/pull/444 I missed 2 places that needed the ruby version set

### CODE REVIEW

**Review by sebastiancaraballo — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -30,6 +30,7 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           bundler-cache: true
+          ruby-version: '3.2.2'
```
Comment (santib) [inline .github/workflows/ci.yml:33]: but this wasn't necessary before, right? why is it needed now?

Comment (santib) [reply .github/workflows/ci.yml:33]: image

I think it's automatically set with the `.ruby-version`

Comment (jpascual1994) [reply .github/workflows/ci.yml:33]: It's not needed, by default, it uses the `. ruby-version `

Comment (guillermoap) [reply .github/workflows/ci.yml:33]: I'll remove it from all places and check which version it's using

Comment (guillermoap) [reply .github/workflows/ci.yml]: hadn't refreshed, you are both right

**Pull request closed without merging**